docs/en/setup/configuration/process_discovery/scanner.md

```diff
@@ -0,0 +1,97 @@
+# Linux Process Detector
+
+The Linux process scanner could detect any process in the Linux with finders. It has two mode:
+1. **REGEX**: could define a regex to filter which processes declare to monitor.
+2. **AGENT_SENSOR**: scan recent active agent which have process status hook.
```

Comment:
I think we need another example for this too. You need to ref to there from go2sky doc.

Reply:
Add an example, I will create other PR to link this official documentation to go2sky.